examples/vite-react-18-suspense-waterfall/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```

Comment:
How about changing the `<title>` if user can access it?

Reply:
Could you update this branch as you want please? 🙇

Reply:
```suggestion
    <title>Suspensive - Example Waterfall with Suspense</title>
```
🤔 How about this??

Reply:
Cool!